tests/testutil/router.go

```diff
@@ -62,6 +66,40 @@ func (tr *TestRoute) GetEndpoint() string {
 	return response
 }
 
+// GetKubeEndpoints performs an http.get against the route's full domain and all paths on it
+// Returns responses if status code is 200 for all of them, otherwise it errors out
+func (tr *TestRoute) GetKubeEndpoints() []string {
```

Comment:
As part of changes in different PR, rename this to `GetKubeEndpointResponse`, and `GetEndpoint` to `GetEndpointResponse`. Need to differentiate from methods that return just a URL.